testing/deploy_kubeflow.py

```diff
@@ -56,6 +57,8 @@ def deploy_kubeflow(test_case):
   # TODO(jlewi): We don't need to generate a core component if we are
   # just deploying TFServing. Might be better to refactor this code.
   # Deploy Kubeflow
+  print("Generate operators.")
```

Comment:
Is this script called by kfctl_test.jsonnet?

I think kfctl_test.jsonnet is just calling kfctl.sh and not using these script..

Reply:
No, this is called by the e2e workflow test.